test/unit/pages/patients.test.js

```diff
@@ -154,6 +154,30 @@ describe('Patients', () => {
       expect(nextProps.showWelcomeMessage.callCount).to.equal(0);
     });
 
+    it('should not redirect to patient data when justLogged query param is set and only one patient available and no invites, but user is a clinic', () => {
```

Comment:
Just a tiny typo here in the spec `justLogged` instead of `justLoggedIn` if you feel like fixing. Thanks for adding the test!